feat: extension MaxAliasesLimiter (#2726)
strawberry-graphql/strawberry: RELEASE.md

Release type: minor

> This PR adds a MaxAliasesLimiter extensions which limit the number of aliases in a GraphQL document.

Reviewer: ```suggestion
This PR adds a MaxAliasesLimiter extensions which limits the number of aliases in a GraphQL document.
```

Release type: minor

> Extension MaxAliasesLimiter
> Add a validator to limit the maximum number of aliases in a GraphQL document.

Reviewer: ```suggestion
This PR adds a MaxAliasesLimiter extensions which limit the number of aliases in a GraphQL document.
```
